nilearn/_utils/docs.py

```diff
@@ -504,10 +504,17 @@ def custom_function(vertices):
 
 # view
 docdict['view'] = """
-view : {'lateral', 'medial', 'dorsal', 'ventral',\
-        'anterior', 'posterior'}, optional
+view : either a string or pair of floats, optional.
+        If a string, must be in {'lateral', 'medial', 'dorsal', 'ventral',\
+        'anterior', 'posterior'}.
+        If a sequence, must be a pair (elev, azim) of float
+        angles in degrees that will manually set a custom view.
+        E.g., view=[270.0, 90.0] or view=(0.0, -180.0). Note that this
+        manual option is only available when the plotting engine is matplotlib.
```

Comment:
I am concerned about adding options that only work for one of the plotting backends. could we have a conversion from elevation, azimuth to plotly's eye, up, center view specification (or the other way around), so that the view can be specified in the same way(s) for both?

Reply:
I was unsure whether it would be worth it, given that the plotly engine is still experimental (if I understand correctly).

If you think it is worth prioritizing this conversion logic so that either view can be used with either engine, then I will look into it (with the caveat that it might take a little while because I have not yet put in the time to understand the (eye, up, center) convention.

Reply:
sorry I replied by email but it seems that got lost:

> I was unsure whether it would be worth it, given that the plotly engine is still experimental (if I understand correctly).
> If you think it is worth prioritizing this conversion logic so that either view can be used with either engine, then I will look into it (with the caveat that it might
> take a little while because I have not yet put in the time to understand the (eye, up, center) convention.

That makes sense. Still I think it could be worth comparing conventions
and figuring out if a (approximate) conversion in either or both
directions is possible, and use the one that is easiest to convert to
the other for the new parameter.

If it's hard we can leave the plotly part for another PR, my hesitation
is just that then we may take a long time to get around to it and the
interface of that function will be more complex because of the
discrepency between the plotly and matplotlib engines

Reply:
If it's  not too much of a hassle, it would be great indeed to have the plotly version altogether.
Thx again for your contribution !